src/components/core/core.js

```diff
@@ -184,19 +196,17 @@ export default class Core extends UIObject {
     this.options.mimeType = mimeType
     sources = sources && sources.constructor === Array ? sources : [sources]
     this.containers.forEach((container) => container.destroy())
-    this.mediaControl.container = null
+    this.mediaControl && (this.mediaControl.container = null)
     this.containerFactory.options = $.extend(this.options, { sources })
-    this.containerFactory.createContainers()
-      .then((containers) => this.setupContainers(containers))
-      .then((containers) => this.resolveOnContainersReady(containers))
+    this.prepareContainers()
   }
 
   destroy() {
     this.disableResizeObserver()
     this.containers.forEach((container) => container.destroy())
     this.plugins.forEach((plugin) => plugin.destroy())
     this.$el.remove()
-    this.mediaControl.destroy()
+    this.mediaControl && this.mediaControl.destroy()
```

Comment:
This `destroy` call doesn't make sense anymore since the `MediaControl` is a `UICorePlugin` and the plugins are destroyed in the previous `forEach` loop.